src/policy_provider/SamplePolicyProvider.c

```diff
@@ -68,18 +68,34 @@ static bool BSLP_PolicyRule_IsConsistent(const BSLP_PolicyRule_t *self)
 static uint64_t get_target_block_id(const BSL_BundleRef_t *bundle, uint64_t target_block_type)
```

Comment:
Not a comment on this change exactly, but what does this do when there are multiple blocks of this type? Is it explained in the header API docs?